pkg/kubelet/dockershim/network/kubenet/kubenet_linux.go

```diff
@@ -686,3 +845,57 @@ func (plugin *kubenetNetworkPlugin) disableContainerDAD(id kubecontainer.Contain
 	}
 	return nil
 }
+
+// given a n cidrs assigned to nodes,
+// create bridge configuration that conforms to them
+func (plugin *kubenetNetworkPlugin) getRangesConfig() string {
```

Comment:
TODO would be to just create our Go struct (that embeds cnitypes.NetConf) and marshel to JSON rather than build JSON ourselves.

Reply:
I actually looked into this (got annoyed), then i discovered that the spec does not really publish any consumable types. So this has to be by hand, for later..

Reply:
@khenidak this is plugin-specific configuration for the 'host-local' plugin. So it's not in the CNI spec because it's config for a specific plugin that kubenet delegates to. You want IPAMConfig from here: https://github.com/containernetworking/plugins/blob/master/plugins/ipam/host-local/backend/allocator/config.go